next-frontend/src/components/results/ResultsTable.tsx

```diff
@@ -27,67 +27,71 @@ export function ResultsTable({
   const solveCount = event.recommendedFormat.expected_solve_count;
 
   return (
-    <Table.Root>
-      <Table.Header>
-        <Table.Row>
-          <Table.ColumnHeader>#</Table.ColumnHeader>
-          {isAdmin && <Table.ColumnHeader>Edit</Table.ColumnHeader>}
-          <Table.ColumnHeader>Competitor</Table.ColumnHeader>
-          <Table.ColumnHeader>Best</Table.ColumnHeader>
-          <Table.ColumnHeader>Average</Table.ColumnHeader>
-          <Table.ColumnHeader>Representing</Table.ColumnHeader>
-          <Table.ColumnHeader colSpan={solveCount} textAlign="left">
-            Solves
-          </Table.ColumnHeader>
-        </Table.Row>
-      </Table.Header>
+    <Table.ScrollArea rounded="md" maxW="full">
+      <Table.Root>
+        <Table.Header>
+          <Table.Row>
+            <Table.ColumnHeader>#</Table.ColumnHeader>
+            {isAdmin && <Table.ColumnHeader>Edit</Table.ColumnHeader>}
+            <Table.ColumnHeader>Competitor</Table.ColumnHeader>
+            <Table.ColumnHeader>Best</Table.ColumnHeader>
+            {solveCount > 2 && <Table.ColumnHeader>Average</Table.ColumnHeader>}
```

Comment:
I feel like `average != 0` is the semantically more sound check

Reply:
If you don't have any concrete results that you can check in the header, do some `listOfResults.some()` shenanigans to see if _any_ result in your currently displayed dataset has an average

Reply:
(Just for reference: There are competitions that hold more than one MultiBLD attempt, i.e. they use `Best of 2` or `Best of 3` format. These will have an `expected_solve_count` larger than 1 but still no average to be displayed)